build-aux/m4/bitcoin_qt.m4

```diff
@@ -342,6 +342,8 @@ AC_DEFUN([_BITCOIN_QT_FIND_STATIC_PLUGINS],[
        elif test x$TARGET_OS = xdarwin; then
          PKG_CHECK_MODULES([QTPRINT], [Qt5PrintSupport], [QT_LIBS="$QTPRINT_LIBS $QT_LIBS"])
        fi
+     else
+       QT_LIBS="-lQt5PlatformSupport $QT_LIBS"
```

Comment:
I believe this hasn't been necessary in the past because qt assumed more
about win builds than other environments. If that's the case, and it's
needed now, we'll need to make sure we only check for this for 5.6+.